src/main/java/Task.java

```diff
@@ -0,0 +1,10 @@
+public class Task {
+    public int status;
+    public int category;
+    public String command;
+    Task(int category, int status, String command){
+        this.category = category;
+        this.status = status;
```

Comment:
The status of the task is always set to 0 when you initialize a new Task object, so the constructor may not have to take in a parameter for status and this.status = 0; could be used instead?